Packs/MicrosoftGraphIdentityandAccess/Integrations/MicrosoftGraphIdentityandAccess/MicrosoftGraphIdentityandAccess_description.md

```diff
@@ -9,6 +9,15 @@ To connect to Microsoft Graph Identity & Access using either Cortex XSOAR Graph
 
 At the end of the process you'll see a message that you've logged in successfully.
 
+### Client Credentials Flow
```

Comment:
please change the `## Authorization` to `### Device Code Flow`
and move the `### Client Credentials Flow` to be before the `#### Required Permissions`